tests/static-checks.sh

```diff
@@ -1119,7 +1143,7 @@ static_check_dockerfiles()
 		files=$(echo "$files_status" | awk '{print $NF}')
         fi
 
-        [ -z "$files" ] && info "No Dockerfiles to check" && return 0
+        [ -z "$files" ] && info "No Dockerfiles to check" && popd && return 0
 
 	# As of this writing hadolint is only distributed for x86_64
 	if [ "$(uname -m)" != "x86_64" ]; then
```

Comment:
This branch lacks a `popd` before `return 0`

Reply:
thanks, updated